src/sdk/p2p/peerconnection-channel.js

```diff
@@ -416,6 +416,16 @@ class P2PPeerConnectionChannel extends EventDispatcher {
   _onOffer(sdp) {
     Logger.debug('About to set remote description. Signaling state: ' +
       this._pc.signalingState);
+    if (this._pc.signalingState !== 'stable') {
+      if (this._isPolitePeer) {
+        // Rollback.
+        Logger.debug('Rollback.');
+        this._pc.setLocalDescription();
+      } else {
+        // Ignore this offer.
```

Comment:
Add log here for debugging purpose.

Reply:
Fixed. Removed comments and added Logger.debug().